### ISSUE TITLE
[iptorrents] Your cookie did not work: The operation was canceled. (Config)

### ISSUE THREAD

**Issue by disasterpiece9000** [Jackett/Jackett#7852]
### Environment

**OS**: Ubuntu 19.10

**Jackett Version**: 0.14.253.0

**Last Working Jackett Version**: Unknown

**Are you using a proxy or VPN?** No

### Description

Previously I had Jackett working fine with IPT. After rebooting my server to apply some updates, I can no longer connect to IPT with Jackett. I assume that Jacket updated when I restarted the service, but I am unaware of what version I was on previously. 

### Logged Error Messages

> System.Exception: Your cookie did not work: The operation was canceled.
   at Jackett.Common.Indexers.IPTorrents.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/IPTorrents.cs:line 160
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 26

**Comment by ilike2burnthing** [comment]
Presumably something to do with https://github.com/Jackett/Jackett/pull/7294 if you were on a build earlier than v0.13.173.

Have you tried updating your cookie? Or removing the indexer and adding it again?

**Comment by disasterpiece9000** [comment]
I re-logged into IPT and tried using that cookie (which was different that before) and I have removed the indexer. Neither of those steps worked :(

**Comment by ilike2burnthing** [comment]
That is everything after `cookie:` (looks like it should be, just making sure)? If so, remove the indexer, and then make sure that iptorrents.json and iptorrents.json.bak (I assume that will be the name, I don't have an account) are no longer in the Indexers folder, or delete them if they are. Then restart Jackett and try adding IPTorrents again from scratch (new cookie and all). 🤞

**Comment by disasterpiece9000** [comment]
I don't have the indexer listed in Jackett because when I put in my cookie info and press Okay it just spins forever and eventually throws an error code. I also checked the Indexers folder and it's empty.

Something weird is happening with my cookies though. I just went to IPT again to get cookies again and the info it returned was different (as in some sections were missing). So I opened an incognitio window with no extensions enabled, logged in again, and got the same thing where some sections were missing. Here's the cookie format it gave me this time:

> __cfduid=randomNumbersAndLetters; cf_clearance=randomNumbersAndLetters; uid=randomNumbers; pass=randomNumbersAndLetters

**Comment by ilike2burnthing** [comment]
You need the Cloudflare UID and clearance too, everything after `cookie:`.

Have you tried using one of the proxies in the configuration? You'd have to use the cookie from the corresponding site.

Do you have any other indexers working? It's a bit odd that the folder would have nothing in it at all if so. If not, see if even a public tracker will work for you.

If other indexers work fine then enable enhanced logging (at the bottom Jackett's page), apply server settings, find and delete or rename log.txt, and try to log into IPT again. When it fails, copy the new log file elsewhere, edit out all personal information and upload it here.

**Comment by disasterpiece9000** [comment]
Sorry, I just got time again to look into this issue. I was able to add a public tracker with no issues. Here is the verbose log from when I try to authenticate with IPT:

[log.txt](https://github.com/Jackett/Jackett/files/4399560/log.txt)

I will try with one of the other domains and edit this response if something different occurs.

**Comment by ilike2burnthing** [comment]
Don't worry, it's a bit of a hectic world at the moment.

Can you try a few more public trackers, or a proxy of 1337x? I know for me 1337x is blocked by my ISP, just want to make sure that isn't what is happening here. Don't worry about logs for them.

On that note, were you able to see if any of the IPT proxies worked for you?

**Comment by RoloSoze** [comment]
@disasterpiece9000 Is this still an issue?

**Comment by 0xW1sKy** [comment]
@RoloSoze looks like this still is an issue as i'm encountering it as well.

**Comment by disasterpiece9000** [comment]
Sorry for the spotty replies. Yes, I am still having this issue, but now I am away from my computer so I won't be able to troubleshoot. If I pick this back up when I get home in a month or so then I may open another issue. Best of luck @0xW1sKy! You might want to create a new issue and reference this one since this has been marked as closed.

**Comment by RoloSoze** [comment]
> Best of luck @0xW1sKy! You might want to create a new issue and reference this one since this has been marked as closed.

I just reopened it. @ngosang Are you able to help? @cadatoiva I just sent you an invite in case you have enough time to look at this issue.

**Comment by ilike2burnthing** [comment]
~I'm going to reopen this~ Rolo beat me to it again 😋 

@0xW1sKy if you can answer my questions above, we'll pick up the issue

@disasterpiece9000 if you like you can unsubscribe from the notifications, and come back when you have free time again

**Comment by prior-l** [comment]
I'm also having this issue (and am a dev myself) happy to provide any logs that are needed.

My Jackett is working with other public and private trackers but I get nearly the same error
```
System.Exception: Your cookie did not work: The operation was canceled.
   at Jackett.Common.Indexers.IPTorrents.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/IPTorrents.cs:line 157
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70
```

**Comment by ngosang** [comment]
Even if the message say `Your cookie did not work:` the real problem is `The operation was canceled.` It's a network problem. This could be caused by the proxy, the vpn or the Mono version of Jackett. IPT is working fine for me. Try to disable vpn/proxy and install .NetCore version of Jackket if you can.

**Comment by prior-l** [comment]
I fixed this for me by disabling ipv6 on my synology.

**Comment by 0xW1sKy** [comment]
not sure what changed. but its working now for me...
Maybe was an issue on the IPT side?

**Comment by RoloSoze** [comment]
Alright, I'll close this ticket since it seems that it's not an issue anymore. If someone is still having this problem, ping me and I'll reopen it. Otherwise, @disasterpiece9000 feel free to reopen it if it's still a problem when you get the chance to test it.

**Comment by abyssalsmoke** [comment]
I'm now having this issue, Was working fine yesterday. 
Now I cant get the cookie to work. Has this come back for anyone else? 
![image](https://user-images.githubusercontent.com/40474693/127215985-e1303dab-4f79-40b7-8a45-e1fa049040ff.png)

**Comment by ilike2burnthing** [comment]
Your connection to the site is timing out, which is different to what those here were experiencing. Try one of the other site links and pull a new cookie from it. If you continue to have problems, open a new issue and provide the information requested.
